library/cool/moebius-boundary.red

```diff
@@ -79,24 +81,14 @@ def s1→moebius-boundary→s1 :
   | loop i → λ j → s1→moebius-boundary→s1/loop i j
   ]
 
-quit
+def test (i : 𝕀) : moebius-boundary =
+  s1→moebius-boundary (loop i)
 
--- there is an invalid fhcom in the middle?!
```

Comment:
I checked and this no longer occurs; I guess we fixed that bug somehow.

Reply:
I fixed the bug but the new updates on this file are stacked up in another PR (which was then blocked by another bug).

Reply:
@favonia Ah, ok.